provider/resource_service.go

```diff
@@ -588,47 +557,10 @@ func serviceModelFromBase(ctx context.Context, b *models.Base) (m ServiceState,
 				}
 			}
 
-			/*kpiTF.CustomThresholds = []*CustomThresholdState{}
-			if kpiTF.ThresholdTemplateID.ValueString() == "" {
-				kpiTF.ThresholdTemplateID = types.StringNull()
-				if kpi["adaptive_thresholds_is_enabled"].(bool) || kpi["time_variate_thresholds"].(bool) {
-					diags.AddWarning("Unsupported thresholding",
-						fmt.Sprintf("Custom threshold support only static non-time-variate thresholds: serviceId=%s kpiId=%s. Fallback to default", b.RESTKey, kpiTF.ID))
-				} else {
-					customThreshold := &CustomThresholdState{}
-
-					unpackThresholds := func(key string) (thresholds *ThresholdSettingModelv2) {
-						thresholds = &ThresholdSettingModelv2{}
-						diags = append(diags, marshalBasicTypesByTag("json", kpi[key].(map[string]interface{}), thresholds)...)
-						thresholds.ThresholdLevels = []*KpiThresholdLevelModel{}
-
-						if itsiThresholds, ok := kpi[key].(map[string]interface{}); ok {
-							levels, err := unpackSlice[map[string]interface{}](itsiThresholds["thresholdLevels"])
-							if err != nil {
-								diags.AddError("Unable to unpack custom KPIs levels from service model", err.Error())
-								return
-							}
-
-							for _, level := range levels {
-								levelTF := &KpiThresholdLevelModel{}
-								diags = append(diags, marshalBasicTypesByTag("json", level, levelTF)...)
-
-								thresholds.ThresholdLevels = append(thresholds.ThresholdLevels, levelTF)
-							}
-						}
-						return
-					}
-
-					customThreshold.AggregateThresholds = []*ThresholdSettingModelv2{unpackThresholds("aggregate_thresholds")}
-					customThreshold.EntityThresholds = []*ThresholdSettingModelv2{unpackThresholds("entity_thresholds")}
-					kpiTF.CustomThresholds = append(kpiTF.CustomThresholds, customThreshold)
-				}
-			}*/
-
-			m.KPIs = append(m.KPIs, kpiTF)
+			m.KPIs = append(m.KPIs, *kpiTF)
```

Comment:
probably would be better to define `kpiTF` as value, i.e. `kpiTF := KpiState{}` instead of `kpiTF := &KpiState{}`, so that you don't have to dereference it here..